beacon_node/execution_layer/src/test_utils/handle_rpc.rs

```diff
@@ -149,14 +171,15 @@ pub async fn handle_rpc<T: EthSpec>(
             Ok(serde_json::to_value(JsonPayloadStatusV1::from(response)).unwrap())
         }
         ENGINE_GET_PAYLOAD_V1 | ENGINE_GET_PAYLOAD_V2 => {
-            let request: JsonPayloadIdRequest = get_param(params, 0)?;
+            let request: JsonPayloadIdRequest =
+                get_param(params, 0).map_err(|s| (s, BAD_PARAMS_ERROR_CODE))?;
             let id = request.into();
 
             let response = ctx
                 .execution_block_generator
                 .write()
                 .get_payload(&id)
-                .ok_or_else(|| format!("no payload for id {:?}", id))?;
+                .ok_or_else(|| (format!("no payload for id {:?}", id), -38001))?;
```

Comment:
May as well add a const for this too. Maybe `UNKNOWN_PAYLOAD_ERROR_CODE`?

https://github.com/ethereum/execution-apis/blob/main/src/engine/common.md#errors

We'll have a whole EL in here soon enough ;)

Reply:
Yeah the reason I didn't use `const`s here is because `go-ethereum` doesn't have them in their code & I couldn't come up with a consistent naming scheme lol. But I'll make them `const`s.